instrumentation/opentelemetry-instrumentation-fastapi/pyproject.toml

```diff
@@ -34,7 +34,8 @@ dependencies = [
 
 [project.optional-dependencies]
 instruments = [
-  "fastapi ~= 0.58",
+    "fastapi >= 0.58.0",
```

Comment:
I might've missed the discussion on this but why change from ~= to >=?

Reply:
Ah, after reading the doc and my commit history, I found I wrongly changed `fastapi ~= 0.58` to `fastapi ~= 0.58.0`, which lead to tox test fail. Then I thought the 0.58.0 is wrong, bcz it just support 0.58.X. So I changed it to `>=0.58.0`.

But in fact, `~=0.58` could support any version lower than 1.0, and current newest fastapi version is 0.111.1. So for now, `~=0.58` and `>=0.58.0` both works.  

I think to be more cautious and ensure compatibility within a known range, we should keep `~=0.58`

Reply:
tks for pointing this out!